tests/unit/client-eviction.tcl

```diff
@@ -441,11 +445,12 @@ start_server {} {
     test "evict clients in right order (large to small)" {
         # Note that each size step needs to be at least x2 larger than previous step 
         # because of how the client-eviction size bucktting works
-        set sizes [list 100000 [mb 1] [mb 3]]
+        set sizes [list [kb 128] [mb 1] [mb 3]]
         set clients_per_size 3
         r client setname control
         r client no-evict on
         r config set maxmemory-clients 0
+        r debug replybuffer-peak-reset-time never
```

Comment:
Yes that was the only test that failed.

Reply:
i see my report indicated a few other tests failed:
https://github.com/redis/redis/pull/9822#issuecomment-1040675854
please look into this and let me know when done.

Reply:
> i see my report indicated a few other tests failed: [#9822 (comment)](https://github.com/redis/redis/pull/9822#issuecomment-1040675854) please look into this and let me know when done.

1. The error in "evict clients in right order" was fixed. the issue was related to the fact that the last clients was not freed due to the memory optimization (it was reducing memory under 128K)

2. the error in replybufsize test was as a result of using libc malloc and not jmalloc. I made several fixes there and is should not fail.

3. the third error (Use takeover to bring slaves back: FAILED: caught an error in the test ERR You should send CLUSTER FAILOVER to a replica
ERR You should send CLUSTER FAILOVER to a replica) 
I was unable to reproduce again when I run the daily on a private branch. but In case the upcoming daily will fail I can look into it...

Reply:
what changes did you make for the `replybufsize.tcl` failures?